blog/_posts/2020-01-14-parallel-updates.md

```diff
@@ -0,0 +1,94 @@
+---
+slug: parallel-updates
+date: 2020-01-14
+author: maia
```

Comment:
fyi i put your name on it because i stole most of the language from your draft, hope that's ok

Reply:
we can also play with what joint-authors might look like, which we'll have to do eventually